CMakeLists.txt

```diff
@@ -45,10 +45,14 @@ ELSE()
 ENDIF()
 
 SET(APP_VERSION 3.0.1)
-SET(APP_BUILD "-dev")
+IF(DEFINED ENV{BUILD} AND NOT $ENV{BUILD} STREQUAL "")
+  SET(APP_BUILD $ENV{BUILD})
```

Comment:
Probably should be something like `MUDLET_BUILD_VERSION` or `MUDLET_VERSION_BUILD` the word `BUILD` is too common for an env variable.